core/src/main/scala/akka/kafka/ConsumerSettings.scala

```diff
@@ -331,6 +336,21 @@ class ConsumerSettings[K, V] @InternalApi private[kafka] (
   def withStopTimeout(stopTimeout: java.time.Duration): ConsumerSettings[K, V] =
     copy(stopTimeout = stopTimeout.asScala)
 
+  /**
+   * The timeout waiting for the transactional partitioned streams that close
+   * on every rebalance when revoking the partitions.
+   */
+  def withTransactionalStreamStopTimeout(transactionalStreamStopTimeout: FiniteDuration): ConsumerSettings[K, V] =
```

Comment:
I recommend renaming these builder methods replacing `transactional` with `eos` to remain consistent with `ProducerSettings`.